packages/compass-e2e-tests/helpers/commands/set-ace-value.js

```diff
@@ -1,9 +1,28 @@
+const debug = require('debug')('compass-e2e-tests').extend('set-ace-value');
+
+const FOCUS_ELEMENT = 'ace_text-input';
+
 const META = process.platform === 'darwin' ? 'Meta' : 'Control';
 
 module.exports = function (app) {
   return async function setAceValue(selector, value) {
     const { client } = app;
     await client.clickVisible(`${selector} .ace_scroller`);
+
+    // make sure the right element is focused before we continue
```

Comment:
This is the flake that looks like a subsequent timeout on macos in CI:

 ![screenshot-Smoke-tests-Aggregations-tab-supports-creating-an-aggregation](https://user-images.githubusercontent.com/69737/135307289-030195d1-fc96-479d-a2ec-67b564c7ad09.png)


Hopefully this code helps.

Reply:
Trying some more things...